adapters/beachfront/beachfront.go

```diff
@@ -437,10 +443,11 @@ func getVideoRequests(request *openrtb.BidRequest) ([]beachfrontVideoRequest, []
 			}
 		}
 
-		if bfReqs[i].Request.Device != nil && bfReqs[i].Request.Device.DeviceType == 0 {
+		if deviceCopy != nil && deviceCopy.DeviceType == 0 {
```

Comment:
It's not possible for `deviceCopy` to be `nil`.

Reply:
Yep, I noticed, left the check anyways but I can remove

Reply:
Cool. Might as well clean that up while we're here. Likely left over from an older implementation.